src/OrchardCore.Modules/OrchardCore.Setup/Startup.cs

```diff
@@ -21,8 +22,8 @@ public Startup(IShellConfiguration shellConfiguration)
         {
             var configurationSection = shellConfiguration.GetSection("OrchardCore.Setup");
 
-            _defaultCulture = configurationSection["DefaultCulture"];
-            _supportedCultures = configurationSection.GetSection("SupportedCultures").Get<string[]>();
+            _defaultCulture = configurationSection["DefaultCulture"] ?? "";
+            _supportedCultures = configurationSection.GetSection("SupportedCultures").Get<List<string>>().ToArray();
```

Comment:
Here one thing I noticed is the fact that we can't add an invariant culture as a supported culture. I think it is not necessary though.